src/Model/listmodel.js

```diff
@@ -1,7 +1,21 @@
 const Item = require('./itemmodel')
 
 class List extends Item {
+  constructor (title, active) {
+    super(title)
+    this.active = active
+    this.tasks = []
+  }
 
+  getIdsByStatus (tasks, status) {
+    const statusTasksIds = []
+    tasks.forEach(task => {
+      if (task.done === status) {
```

Comment:
Ich finde `task.done` gut. Allerdings hast du in item mit `item.getTitle` angefangen. Das naming sollte einheitlich sein, also entweder `task.getDone` und `item.getTitle` oder `task.done` und `item.title`. Mir persönlich gefällt letzteres besser. 

Ich meine auch dass es eher üblich ist ohne get/set im naming zu arbeiten wenn du get und set in den Klassen einsetzt. Hier ein Beispiel: https://javascript.info/property-accessors#getters-and-setters

Reply:
Habe es umgeschrieben, wie du es vorgeschlagen habe. Bitte schau dir den Code und den Link im Kommentar dazu richtig an!